docs/content/concepts/configuration/config-schema.mdx

```diff
@@ -5,91 +5,119 @@ description: Job run configuration allows providing parameters to jobs at the ti
+</TabItem>
+<TabItem name="Using ops and jobs">
 
-```python file=/concepts/configuration/configurable_op_asset_resource.py startafter=start endbefore=end
-@op(config_schema={"person_name": str})
-def op_using_config(context):
-    return f'hello {context.op_config["person_name"]}'
+#### Using ops
 
+Here, we define a subclass of <PyObject object="Config"/> holding a single string value representing the name of a user. We can access the config through the `config` parameter in the op body.
 
-@asset(config_schema={"person_name": str})
-def asset_using_config(context):
-    # Note how asset config is also accessed with context.op_config
-    return f'hello {context.op_config["person_name"]}'
+```python file=/guides/dagster/pythonic_config/pythonic_config.py startafter=start_basic_op_config endbefore=end_basic_op_config dedent=4
+from dagster import op, Config
 
+class MyOpConfig(Config):
+    person_name: str
 
-@resource(config_schema={"url": str})
-def resource_using_config(context):
-    return MyDatabaseConnection(context.resource_config["url"])
+@op
+def print_greeting(config: MyOpConfig):
+    print(f"hello {config.person_name}")  # noqa: T201
 ```
 
-<Note>
-  It is technically possible to access <code>context.op_config</code> inside ops
-  (not assets) without defining a <code>config_schema</code>. However, this is
-  not recommended.
-</Note>
+</TabItem>
+</TabGroup>
 
 You can also build config into jobs, as described in [the Jobs documentation](/concepts/ops-jobs-graphs/jobs#configuring-jobs).
 
+### Defining and accessing Pythonic configuration for a resource
+
+Configurable parameters for a resource are defined by specifying attributes for a resource class, which subclasses <PyObject object="ConfigurableResource"/>. The below resource defines a configurable connection URL, which can be accessed in any methods defined on the resource.
+
+```python file=/guides/dagster/pythonic_config/pythonic_config.py startafter=start_basic_resource_config endbefore=end_basic_resource_config dedent=4
+from dagster import op, ConfigurableResource
+
+class MyDatabaseResource(ConfigurableResource):
+    connection_url: str
+
+    def query(self, query: str):
+        return get_engine(self.connection_url).execute(query)
+```
+
+For more information on using resources, see [the Pythonic resources guide](/guides/dagster/pythonic-resources).
+
 ---
 
 ## Specifying runtime configuration
 
-If you want to execute `op_using_config` or materialize `asset_using_config`, we'll need to provide values for the parameters specified in `config_schema`. How we provide these values depends on the interface we are using:
+To execute a job or materialize an asset that specifies config, you'll need to provide values for its parameters. How we provide these values depends on the interface we are using:
 
 - [Python](#python)
 - [Dagit](#dagit)
 - [Command line](#command-line)
```

Comment:
Since we're in here, would you mind putting these three sections into tabs? This is a really good place for that, as users will likely only need to choose one of these.